nodejs/app.js

```diff
@@ -0,0 +1,36 @@
+import createShipment from "./shipment.js";
+import getShippingLabel from "./labels.js";
+import dotenv from "dotenv";
+
+const inti = async () => {
+  const authorization = process.env.AUTHORIZATION;
+  const clientId = process.env.CLIENT_ID;
+
+  console.log("\nCreate delivery 📦");
+  var shipment = await createShipment(authorization, clientId);
+  var shipmentNumber = shipment.shipment_number;
+  var trackingNumber = shipment.deliveries[0].tracking_number;
+  console.log(">> Done! 🌟");
+  console.log(`>> shipment_number: ${shipmentNumber}`);
+  console.log(`>> tracking_number: ${trackingNumber}`);
+
+  console.log("\nGet shipping label 🏷️");
+  await getShippingLabel(authorization, trackingNumber);
+  console.log(">> Done! 🌟");
+  console.log(">> zpl saved in current folder");
+
+  console.log("\nBye bye");
+};
+
+(async () => {
```

Comment:
Top level await is available from Node v14 so you don't need to wrap this in an async IFEE

Reply:
Of course, let's change.